### DISCUSSION TITLE
求教Uploader 自定义预览样式怎么实现!!万分感谢!!

### DISCUSSION THREAD

**Discussion by JasonM0321** [youzan/vant-weapp#5942] [Q&A]
![uploader截图](https://github.com/user-attachments/assets/67073676-8ff3-47d2-92b5-9a49b15d324f)
求教自定义预览样式中这个底部的文字是怎么添加 上去的，找了一圈实在没有找到怎么实现。

**Comment by Dagen2369** [comment]
解决了吗，我也在找这个问题
